lua/wikis/commons/Icon/Data.lua

```diff
@@ -12,6 +12,7 @@ return {
 	loss = 'fas fa-times',
 
 	-- Usage: Other Match detail pop up icons
+	matchpagelink = 'fas fa-external-link', -- Should we really use external for this?
```

Comment:
if we have an internal alternative imho preferable

Reply:
it's the icons we have atm, but I think we should look (later) at changing them